### DISCUSSION TITLE
Connecting to MongoDB

### DISCUSSION THREAD

**Discussion by harshdeepsinghin** [salman0ansari/whatsapp-api-nodejs#465] [Q&A] [answered]
getting error while connecting to mongodb

"err":{"type":"MongooseServerSelectionError","message":"connect ECONNREFUSED 127.0.0.1:27017","stack":"MongooseServerSelectionError: connect ECONNREFUSED 127.0.0.1:27017\

**Comment by salman0ansari** [comment, marked as answer]
check mongodb status
`sudo systemctl status mongod`

**Reply by harshdeepsinghin** [reply]
not running,

```Unit mongod.service could not be found.```

**Reply by harshdeepsinghin** [reply]
Thanks brother, its done now!

**Reply by mohdlatif** [reply]
Do I have to create a Mongodb account and  replace the URL in the .env ?
